recipes/_centos.rb

```diff
@@ -0,0 +1,7 @@
+include_recipe 'yum-ius'
+include_recipe 'yum-epel'
+include_recipe 'yum-osuosl'
```

Comment:
Do you really need all three of these repos? It looks to me like you only need ius.